packages/squiggle-lang/benchmark/conversion_tests.ts

```diff
@@ -0,0 +1,46 @@
+import { distributions, generateInt, generateFloatRange } from "./generators";
+import { test, expectEqual } from "./lib";
+
+let checkDistributionSame = (
+  distribution: string,
+  operation: (arg: string) => string
+): void => {
+  expectEqual(
+    operation(distribution),
+    operation(`toPointSet(${distribution})`)
+  );
+  expectEqual(
+    operation(distribution),
+    operation(`toSampleSet(${distribution})`)
+  );
+};
+
+Object.entries(distributions).map(([key, generator]) => {
+  let distribution = generator();
+  test(`mean is the same for ${key} distribution under all distribution types`, () =>
+    checkDistributionSame(distribution, (d: string) => `mean(${d})`));
+
+  test(`cdf is the same for ${key} distribution under all distribution types`, () => {
+    let cdf_value = generateInt();
```

Comment:
Wait; so this generates one randomly chosen cdf_value, for each distribution? This seems weird to me. Either lots get chosen, or we should have like 3-10 for each distribution. (Can leave for future iterations though)